doc/man5/x509v3_config.pod

```diff
@@ -284,10 +284,18 @@ B<access_id;location>, where B<access_id> is an object identifier
 (although only a few values are well-known) and B<location> has the same
 syntax as subject alternative name (except that B<email:copy> is not supported).
 
+Possible values for access_id are B<OCSP> (OCSP responder),
+B<caIssuers> (CA Issuers), 
+B<ad_timestamping> (AD Time Stamping), 
+B<AD_DVCS> (ad dvcs),
+B<caRepository> (CA Repository).
+
 Examples:
 
- authorityInfoAccess = OCSP;URI:http://ocsp.my.host/
+ authorityInfoAccess = OCSP;URI:http://ocsp.my.host/,caIssuers;URI:http://my.ca/ca.cer
 
+ authorityInfoAccess = OCSP;URI:http://ocsp.my.host/
```

Comment:
Arguably we should be using RFC 2606 reserved-for-documentation names, though I won't insist on making that change as part of this PR.

Reply:
Updated x509v3_config.pod examples to example.com & example.org